mw-webapp/src/logic/page404/Page404.module.scss

```diff
@@ -5,7 +5,7 @@
   display: flex;
   justify-content: center;
   margin-top: 50px;
-  font-size: $font-size-large;
+  font-size: $fontSizeLarge;
```

Comment:
It's ok to create mixins for text, let's wait for designer answer

Reply:
Ok, let's wait.
Here I changed kebab case to camel case. Could I change the writing style to camel case or kebab case?